src/injected/visualization/drawer-impl.ts

```diff
@@ -0,0 +1,158 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { FeatureFlagStoreData } from '../../common/types/store-data/feature-flag-store-data';
+import { WindowUtils } from '../../common/window-utils';
+import { ClientUtils } from '../client-utils';
+import { DialogRenderer } from '../dialog-renderer';
+import { AxeResultsWithFrameLevel } from '../frameCommunicators/html-element-axe-results-helper';
+import { HtmlElementAxeResults } from '../scanner-utils';
+import { ShadowUtils } from '../shadow-utils';
+import { BaseDrawer } from './base-drawer';
+import { DrawerInitData } from './drawer';
+import { DrawerUtils } from './drawer-utils';
+import { BoxConfig, DrawerConfiguration, Formatter } from './formatter';
+
+export class DrawerImpl extends BaseDrawer {
```

Comment:
Seeing `DrawerImpl extends BaseDrawer` implies to me that one of the following is probably true:

* There are other `Drawer` implementations, so it'd be nice to use a more descriptive name than `DrawerImpl`
* There aren't any other `Drawer` implementations, so `BaseDrawer` is probably redundant with `DrawerImpl`

Reply:
Make sense. Updated.